src/ui/skills.py

```diff
@@ -10,6 +11,196 @@
 from ui_manager import wait_until_visible, ScreenObjects
 from d2r_image import ocr
 
+class SkillName(str, Enum):
```

Comment:
These names have to map the ones used in the skill templates, right? Maybe a comment would be nice about this kind of hidden dependency.

Reply:
Class docstring has been added to SkillName and HotkeyName